squarelet/email_api/tests/factories.py

```diff
@@ -0,0 +1,17 @@
+# Third Party
+import factory
+from allauth.account.models import EmailAddress
+
+# Squarelet
+from squarelet.users.tests.factories import UserFactory
+
+
+class EmailFactory(factory.django.DjangoModelFactory):
+    email = factory.Sequence(lambda n: f"user-email-{n}@example.com")
+    verified = False
+    primary = factory.Sequence(lambda n: False if n > 0 else True)
```

Comment:
This is not tied to one user, so could make things confusing - I think it might be better to just set this to False and explicitly set to true when needed